### PR TITLE
data: theVatsal3802

### PR DESCRIPTION
**Pull request by theVatsal3802**
Added `Events` Files.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4209"></a>

### CODE REVIEW

File: data/theVatsal3802.json [hunk 1]
```diff
@@ -1,29 +1,68 @@
...
+        "Flutter",
+        "Android",
+        "Machine Learning",
+        "Python",
+        "Quantum Computing"
+    ],
+    "links": [
+        {
+            "name": "Mail ID",
+            "url": "vatsaladhiya248@gmail.com",
```
Comment (Aadarsh805) [inline data/theVatsal3802.json:39]: make it a mailto link, like this `mailto:vatsaladhiya248@gmail.com`

File: data/theVatsal3802.json [hunk 2]
```diff
@@ -1,29 +1,68 @@
...
-      "icon": "FaGithub"
-    }
-  ]
+    "name": "Vatsal Adhiya",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "socials": [
+        {
+            "icon": "SiGmail",
+            "url": "vatsaladhiya248@gmail.com"
```
Comment (Aadarsh805) [inline data/theVatsal3802.json:8]: here as well, make it `mailto:vatsaladhiya248@gmail.com`

Comment (theVatsal3802) [reply data/theVatsal3802.json]: Done

File: data/theVatsal3802.json (hunk 1 above)
Comment (theVatsal3802) [reply data/theVatsal3802.json]: Done

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by Tejender1521** [comment]
Everything looks good to me and ready to be merged.

**Review by raghu-007 — APPROVED** [review]
well done

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to me! 👍 

>**Note**
> Please note that currently LinkFree doesn't support past event, so it will not be displayed on you profile. But soon will. 🙂

**Pull request merged**